src/app.js

```diff
@@ -1 +1,30 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const { question } = require('./question');
+
+const main = async() => {
+  let filename = '';
+  let content = '';
+
+  while (!filename.length) {
+    filename = await question('Filename: ');
+  }
+
+  while (!content.length) {
+    content = await question('Content: ');
+  }
+
+  fs.writeFile(filename, content, (err) => {
+    if (err) {
+      console.log('Error');
+
+      return;
+    }
+
+    console.log('Success');
```

Comment:
For a simple console.log you could use ternary to shorten it a bit

Reply:
I already refactored it to its own function :D let it have a little breathing space